scp/lib/login.dart

```diff
@@ -38,7 +38,7 @@ class _LoginState extends State<Login> {
     var textScaleFactor = MediaQuery.of(context).textScaleFactor;
     countryCodeController.text = countryCode;
     return Scaffold(
-      resizeToAvoidBottomPadding: false,
```

Comment:
![image](https://user-images.githubusercontent.com/27865704/114801559-8e7d0280-9db9-11eb-982c-8a91904e37e8.png)
A diff of 396 lines, all formatter diff
Why does it happen?

Reply:
Everything was not just for formatting, It was basically because we had to wrap the entire widget tree with the future builder as we used shared preference to fetch the semester kind.